client/src/utils/getValueFiatToDisplay.ts

```diff
@@ -47,7 +51,7 @@ export default function getValueFiatToDisplay({
     !cryptoValues[cryptoCurrency][displayCurrency] ||
     !valueCrypto
   ) {
-    return `${prefix}0.00`;
+    return showLessThanZero ? `< ${prefix}0.00` : `${prefix}0.00`;
```

Comment:
I think this ternary here is not required. It's for the loading states only. Or maybe I'm wrong?